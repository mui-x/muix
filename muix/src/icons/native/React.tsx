import createSvgIcon from './lib/create-svg-icon'

export default createSvgIcon(
  'react',
  'React',
  true
)