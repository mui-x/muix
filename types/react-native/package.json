{
  "name": "@types/react-native",
  "version": "0.1.1",
  "description": "@types/react like version of TypeScript definitions for react-native (with 'export as namespace ReactN')",
  "registry": "npm",
  "dependencies": {},
  "files": [ "index.d.ts" ]
}
