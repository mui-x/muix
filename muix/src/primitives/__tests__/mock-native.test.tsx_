import React from 'react'
import { initPlatform, mount } from "reactxx-tests"
import { Text, Animated } from 'react-native'

describe("SWITCH define sheet", () => {
  it("NATIVE", () => {
    initPlatform(false)
    const wrapper = mount(
      <Animated.ScrollView color=''>
        <Text style={{color:'red'}}>Halo world</Text>
      </Animated.ScrollView>)
    expect(wrapper).toMatchSnapshot()
  })
})


