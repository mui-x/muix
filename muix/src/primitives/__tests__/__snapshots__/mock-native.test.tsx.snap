// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SWITCH define sheet NATIVE 1`] = `
<div>
  <div
    data="ANIMATEDVIEW"
    style="background-color: blue;"
  >
    <div
      data="TEXT"
      style="color: red;"
    >
      Hallo world
    </div>
  </div>
</div>
`;
